src/appleseed.studio/mainwindow/renderingsettingswindow.cpp

```diff
@@ -432,19 +435,37 @@ namespace
         Q_OBJECT
 
       public:
-        explicit GeneralSettingsPanel(const Configuration& config, QWidget* parent = nullptr)
+        explicit GeneralSettingsPanel(QWidget* parent = nullptr)
           : RenderSettingsPanel("General Settings", parent)
         {
-            QFormLayout* layout = create_form_layout();
-            container()->setLayout(layout);
+        }
 
+      protected:
+        void create_color_pipeline(QFormLayout* parent)
+        {
             QComboBox* color_pipeline_combobox = create_combobox("spectrum_mode");
             color_pipeline_combobox->setToolTip(m_params_metadata.get_path("spectrum_mode.help"));
             color_pipeline_combobox->addItem("RGB", "rgb");
             color_pipeline_combobox->addItem("Spectral", "spectral");
-            layout->addRow("Color Pipeline:", color_pipeline_combobox);
+            parent->addRow("Color Pipeline:", color_pipeline_combobox);
 
             create_direct_link("spectrum_mode", "spectrum_mode", "rgb");
+        }
+    };
+
+    class FinalGeneralSettingsPanel
+      : public GeneralSettingsPanel
+    {
+        Q_OBJECT
+
+      public:
+        explicit FinalGeneralSettingsPanel(const Configuration& config, QWidget* parent = nullptr)
+            : GeneralSettingsPanel(parent)
```

Comment:
Just two spaces (half indent) here.

This idiosyncracy will go away when we switch to clang-format (after the next release)...